docs/getting-started/integrating-codacy-with-intellij-ides.md

```diff
@@ -0,0 +1,86 @@
+
+-   Sequences of **duplicate code** (clones) introduced by the pull request. These are the same ones you find in the [Quality Duplication tabs](../repositories/pull-requests.md#duplication-tabs) in the Codacy app.
+
+-   Variations in **code complexity** introduced by the pull request.
+
+## Installing the Codacy IntelliJ plugin
+
+!!! info "The Codacy IntelliJ plugin is compatible with IntelliJ IDEA (Ultimate, Community), Android Studio, and Aqua."
+
+1.  Make sure that the repository you’re working on is analyzed by Codacy and that you have a [repository read](../organizations/roles-and-permissions-for-organizations.md) role or higher.
+
+    !!! tip
+        If this is your first time using Codacy, see [how to add and analyze your first repository](./codacy-quickstart.md#adding-your-first-repository).
+
+1.  Install the plugin from the [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/23924-codacy) or through the [plugin settings panel](https://www.jetbrains.com/help/idea/managing-plugins.html) in your IntelliJ IDE.
+
+## Getting pull request quality and coverage data
+
+To see Codacy quality and coverage data for an open pull request, follow these steps:
+
+1.  Open the repository directory in your IntelliJ IDE.
+
+    !!! note
+        If the repository isn't on Codacy yet, [add it to Codacy](../organizations/managing-repositories.md#adding-a-repository) first.
+
+1.  Open the main view by clicking the **Codacy logo** in the left tool window bar.
+
+    ![Codacy main view](./images/codacy-intellij-plugin-sign-in.png)
+
+1.  If you’re not signed in, click the **Sign in** button to authorize IntelliJ on Codacy.
+
+1.  Check out the pull request of interest.
+
+After completing these steps, the main view shows the result of the latest Codacy analysis for the pull request.
+
+## Reviewing pull request issues
+
+In the **Problems tool window**, Codacy displays the same Quality issues you find in the main view and lets you navigate to the exact line of code where the issue was found.
+
+!!! note
+    Code coverage, duplicates, and complexity aren't currently shown in the Problems tool window.
+
+To review Quality issues:
+
+1.  Open the file you want to review.
+
+    You can find the list of files with issues in the main view of the Codacy plugin. Double-click the file name to open it in the editor.
+
+1.  Open the **Problems tool window** (use `Ctrl+6` on Windows/Linux or `Cmd+6` on macOS) and select the **File** tab.
+
+    ![Navigate to a specific issue from the Problems tab](./images/codacy-intellij-plugin-problems-tab.png)
+
+1.  Click the name of the issue you want to review.
+
+1.  Hover over a highlighted issue in the code editor to view available actions and suggested quick fixes, if available.
+
+    For a list of tools that support quick fixes, see [Supported languages and tools](./supported-languages-and-tools.md#supported-languages-and-tools).
```

Comment:
Consider using **suggested fixes** instead to match the column name of the supported tools table.

```suggestion
    For a list of tools that support suggested fixes, see [Supported languages and tools](./supported-languages-and-tools.md#supported-languages-and-tools).
```